examples/services/fabricv4/connections/fcr_2_azure_connection.py

```diff
@@ -0,0 +1,141 @@
+import time
+import os
+
+from examples.services.fabricv4.utils import utils
+from equinix.services import fabricv4 as module
```

Comment:
Don't need the as in import.

Reply:
I think we should add `black` to the GH actions to reduce the need for style feedback in reviews.

Similar action in another Equinix SDK: https://github.com/packethost/packet-python/blob/master/.github/workflows/test.yml#L15-L26